src/containers/stage-header.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import VM from 'scratch-vm';
+import {setZoomed} from '../reducers/zoom';
+
+import {connect} from 'react-redux';
+
+import StageHeaderComponent from '../components/stage-header/stage-header.jsx';
+
+class StageHeader extends React.Component {
+    shouldComponentUpdate (nextProps) {
+        return this.props.width !== nextProps.width ||
+               this.props.height !== nextProps.height ||
+               this.props.isZoomed !== nextProps.isZoomed;
+    }
```

Comment:
I don't believe this will have different behavior than the default `shouldComponentUpdate` (it should rerender whenever the state changes), so this can be removed.

Reply:
I took this out, but it now fails the lint checker. I added an exception for now. Let me know if you want this converted.